stubs/CoreGenericFunctions.phpstub

```diff
@@ -66,7 +66,7 @@ function array_intersect_assoc(array $arr, array $arr2, array ...$arr3)
  * @param array<mixed, TKey> $arr
  * @param array<mixed, TValue> $arr2
  *
- * @return array<TKey, TValue>|false
+ * @return ($arr is non-empty-array ? non-empty-array<TKey, TValue>|false : array<empty, empty>|false)
```

Comment:
@muglug , @orklah , why `array<empty, empty>`? Shouldn't it also be `array<TKey, TValue>`?

Reply:
Yeah, you're right, I assumed the opposite of non-empty-array would be an empty array but it's not true. I'm fixing that